roles/online_docs/templates/mkdocs/docs/apptainer.md

```diff
@@ -0,0 +1,595 @@
+    - [Containers and apptainer](#containers-and-apptainer)
+- [List of commonly used commands](#list-of-commonly-used-commands)
+- [Building Apptainer images](#building-apptainer-images)
+    - [Building Without Elevated Permissions](#building-without-elevated-permissions)
+- [Sandbox](#sandbox)
+    - [Example with Ubuntu 18.04](#example-with-ubuntu-1804)
+    - [Converting sandbox directory to sif file](#converting-sandbox-directory-to-sif-file)
+- [Building image as a regular user](#building-image-as-a-regular-user)
+- [Making an image from the definition file with elevated permissions](#making-an-image-from-the-definition-file-with-elevated-permissions)
+- [Definition file](#definition-file)
+    - [Header](#header)
+    - [Sections](#sections)
+- [Important](#important)
+- [Caching redirect or disable](#caching-redirect-or-disable)
+- [Running Apptainer on Windows or Mac](#running-apptainer-on-windows-or-mac)
+- [Using images from the Docker repository](#using-images-from-the-docker-repository)
+- [Isolated runs](#isolated-runs)
+- [Mounting host folders inside containers](#mounting-host-folders-inside-containers)
+- [Other considerations](#other-considerations)
+    - [Architecture considerations](#architecture-considerations)
+    - [Licence considerations](#licence-considerations)
+- [Environment variables](#environment-variables)
+- [Debug information](#debug-information)
+- [Additional examples](#additional-examples)
+    - [Alpine as regular user build static file via sandbox directory](#alpine-as-regular-user-build-static-file-via-sandbox-directory)
+    - [Simple CentOS 7 example via sandbox](#simple-centos-7-example-via-sandbox)
+
+## About Apptainer
+
+### In short
+
+> Container is an isolated environment that holds the software and its dependencies and configurations. Containers can be run on any machine with compatible container technology.
+Container instance can be run (deployed) from an existing image. Therefore users must first either build (create) or copy an existing image (f.e. pull from a public registry). They can also share their prebuilt images with each other, and running containers from those, they can expect always to get the same results, regardless of the system on which the containers are run.
+> Apptainer is one of the container technology implementations; Docker is, for example, another. They work differently manner and are not fully compatible.  Apptainer is suited for HPC environments but can also run images from the Docker repository.
+> Users can deploy containers, and inside those containers can access the same resources as they can access outside of those containers - nothing more.
+> Container technology described here is focused only on building and deploying Linux containers on Linux host systems.
+
+Documentation is available on the official [Apptainer's documentation website](https://apptainer.org/docs/)
+
+### Containers and apptainer
+
+About Linux containers
```

Comment:
```suggestion
Linux containers
```

Reply:
Note: blank line before list items is required to format lines starting with `*` as list items. Otherwise, you'll get one long paragraph with `*` in between the lines.

Reply:
Noted. Thanks!